hudi-common/src/test/java/org/apache/hudi/common/model/TestOverwriteWithLatestAvroPayload.java

```diff
@@ -98,6 +106,18 @@ public void testDeletedRecord() throws IOException {
 
     assertEquals(payload1.combineAndGetUpdateValue(delRecord1, schema).get(), record1);
     assertFalse(payload2.combineAndGetUpdateValue(record1, schema).isPresent());
+
+    // test userDefinedDeleteField
```

Comment:
we could move this to a private method and so lines 99 tp 108 and 110 to 120 can re-use code